### DISCUSSION TITLE
Need Tutorials

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [Voidware-Prohibited/ALSXT#38] [General]
Personally as someone new to this system, I am hoping someone will start making Video Tuts. I have no clue how to set up upper body animation. Montages fail at this on this system. I have also posted this on the ALS-Refactored site.

**Comment by colorindarkness** [comment]
Hello, yes, there's unfortunately a huge lack of documentation as well. The main reason animations may not work is because they must be assigned to the correct animation slot, and lacks the required animations curves. I am slowly adding Documentation to make it easier to understand and customize. Once I reach Alpha Release I will be prioritizing Documentation
